src/app/pages/main/drivers/drivers.page.html

```diff
@@ -2,24 +2,28 @@
   <mat-card-title>Pilotos</mat-card-title>
 
   <mat-card-content>
-    <mat-table #table [dataSource]="drivers">
+    <mat-table #table [dataSource]="seasonsService.drivers">
       <mat-header-row *matHeaderRowDef="displayedColumns"></mat-header-row>
       <mat-row *matRowDef="let row; columns: displayedColumns;"></mat-row>
 
       <ng-container matColumnDef="Driver">
         <mat-header-cell *matHeaderCellDef> Piloto </mat-header-cell>
         <mat-cell *matCellDef="let driver">
           <img class="flagImg" src="../../../../assets/icons/flags/{{driver.countryCode}}.png" alt="flag">
-          <rm-driver [driverCode]="driver.code"></rm-driver>
+          <rm-driver [driver]="driver"></rm-driver>
         </mat-cell>
       </ng-container>
+      <ng-container matColumnDef="Initial Price">
```

Comment:
(only doubt) Why we need to know what is the initial value and the actual value?

Reply:
I thought it added value... if you disagree we can discuss